frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/SpecificDatePicker.tsx

```diff
@@ -27,7 +28,7 @@ type Props = {
 
 const SpecificDatePicker: React.FC<Props> = props => {
   const onChange = (
-    date?: string | moment.Moment,
+    date?: string | momentTimezone.Moment,
```

Comment:
I don't love types coming from `moment` and `moment-timezone`, but I suppose the libraries are like this. Moving everything to `moment-timezone` might be ill-advised, and a ton of work.